src/app.js

```diff
@@ -274,11 +277,13 @@ export function app(app) {
         var reusableNode = reusableChild[1]
         if (!newKeys[reusableNode.data.key]) {
           removeElement(element, reusableChild[0], reusableNode)
+          recoverNode(reusableNode)
```

Comment:
wasn't sure if this is safe to recoverNode here, since this is where keyed nodes are re-used...